src/database-controller/sdk/index.js

```diff
@@ -0,0 +1,301 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+const { Sequelize, Model } = require('sequelize');
+
+class DatabaseModel {
+  constructor(connectionStr, maxConnection = 10) {
+    const sequelize = new Sequelize(connectionStr, {
+      pool: {
+        max: maxConnection,
+        min: 1,
+      },
+    });
+
+    class Framework extends Model {}
+    Framework.init(
+      {
+        insertedAt: Sequelize.DATE,
+        name: {
+          type: Sequelize.STRING(64),
+          primaryKey: true,
+        },
+        namespace: Sequelize.STRING(64),
```

Comment:
Better to doc table each column

Reply:
Most columns are easy to understand. I document some of them , e.g. `requestSynced`, `apiServerDeleted`, `archived`, `dockerSecretDef`, `configSecretDef`, `priorityClassDef`, `submissionTime`.